requirements-dev.txt

```diff
@@ -11,10 +10,7 @@ numpy
 scipy
 selenium
 pytest-xdist
-pytest-rerunfailures>=8.0; platform_python_implementation != "PyPy"
-pytest-rerunfailures>=8.0,<9.0; platform_python_implementation == "PyPy"
-pytest-faulthandler; platform_python_implementation != "PyPy"
-pytest-faulthandler<2.0; platform_python_implementation == "PyPy"
+pytest-rerunfailures>=8.0
```

Comment:
From the pins, seems like 9.0 was broken for pypy. Not sure if whatever problem happened is now fixed or when it was fixed, but if we pin, I'd go for `>=10`, no?